### ISSUE TITLE
New module version tag

### ISSUE THREAD

**Issue by antoi-ne** [karalabe/hid#54]
Hello,
When go-getting this module, it will by default download the latest tagged version (`v1.0.0`, 2019) which seems not to work on some platforms (tested on macOS 14.2.1, arm64), but works when getting the latest commit.
Could a newer version tag be created so the module works without having to point to a specific commit?
Thanks a lot! :)

**Comment by michealroberts** [comment]
@karalabe Yes, same issue here. Where are the tags? This library, as it currently stands, is not fit for purpose.

**Comment by heyvito** [comment]
Stumbled at the same issue. Had to `go get github.com/karalabe/hid@master` to get the latest version.

**Comment by antoi-ne** [comment]
Pinging @holiman since its been some time, could you tag a new release?
